### DISCUSSION TITLE
Welcome to Mac@IBM Notifications Discussions!

### DISCUSSION THREAD

**Discussion by SMartorelli** [IBM/mac-ibm-notifications#6] [General]
## 👋 Welcome to the IBM Notifier project!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

[Project Wiki](https://github.com/IBM/mac-ibm-notifications/wiki) - [Feature request](https://github.com/IBM/mac-ibm-notifications/issues/new?assignees=&labels=&template=feature_request.md&title=) - [Bug report](https://github.com/IBM/mac-ibm-notifications/issues/new?assignees=&labels=&template=bug_report.md&title=)

**Comment by Kays10** [comment]
How do I use this app with App installers to notify users of new app updates

**Reply by SMartorelli** [reply]
Hi @Kays10 👋 I don’t think that Jamf offers the ability to use a different notification service for updates of the apps deployed using Jamf App Installers. If I’m not wrong, the update notification workflow is solely managed by the Self Service built-in notification system in the Notification Center.
